README.md

```diff
@@ -47,11 +74,11 @@ git clone https://github.com/ares-emulator/ares.git
 From the root of the project directory run:
 
 ```
-make -j4 build=release
+make -j$((`nproc`-1)) build=release
```

Comment:
I'm not so sure about adding an implicit dependency on an sh-like shell and Unix utilities like nproc for this example. Neither is required to build ares.

Reply:
On an unrelated note, this example already isn't matching the build configuration used for official releases (`build=optimized`). I think it would be fine to just let the user pick up the default (currently `optimized`).

Reply:
Reverted to old verbiage for parallel processes. Removed specific build so it accepts default and added section to describe build options if anyone is interested.